### PR TITLE
New OpenGL Cheat Sheet

### PR DESCRIPTION
**Pull request by yasharmaster**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [x] New Instant Answer
  - [x] Cheat Sheets: **`New OpenGL Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
  - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
  - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**
###### Description of changes

What changes does this PR introduce? 
###### Related Issues or Discussions

What issues or discussions (if any) does this PR address? Please provide the issue # or link.
###### People to notify

@mention any relevant people/organizations
@tagawa 

---

Instant Answer Page: https://duck.co/ia/view/opengl_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html):

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=OpenGL Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/opengl_cheat_sheet_mobile.png?7990FNUu9G)

File: share/goodie/cheat_sheets/json/opengl.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+    "id": "opengl_cheat_sheet",
+    "name": "OpenGL",
+    "description": "OpenGL Help",
+    "metadata": {
+        "sourceName": "OpenGL 4.5 Reference Pages",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/opengl.json]: "OpenGL"  would be enough here

File: share/goodie/cheat_sheets/json/opengl.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
+{
+    "id": "opengl_cheat_sheet",
+    "name": "OpenGL",
+    "description": "OpenGL Help",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/opengl.json]: Lets describe about the contents of the cheat sheet. or give a short descpn about opengl

File: share/goodie/cheat_sheets/json/opengl.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    ],
+    "section_order": [
+        "Initializing, Creating and Destroying a Window",
+        "Viewport Transformation",
+        "Basic Drawing Commands",
+        "Describing Points, Lines and Polygons"
+    ],
+    "sections": {
+        "Initializing, Creating and Destroying a Window": [{
+            "val": "Initializes GLUT and processes any command-line arguments. The command line options are dependenton the window system. glutInit() should be called before any other GLUT routine.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/opengl.json]: Lets remove periods at end of val fields

Comment (yasharmaster) [inline share/goodie/cheat_sheets/json/opengl.json]: @jophab done!

**Comment by pjhampton** [comment]
Closing as this looks to have gone stale

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**